### DISCUSSION TITLE
在namespace中没有任何配置时，一直报错could not find config

### DISCUSSION THREAD

**Discussion by small-fresh-meet** [apolloconfig/apollo#4983] [General]
我们项目中默认没有使用application.properties这个namespace，里面没有任何配置。但是控制台会一直报404错误，Could not find config for namespace，同时配置了apollo.refreshInterval=5。虽然知道对业务没有影响，但是一直在打印这个异常日志，如何能一直刷新配置缓存，同时还能把这个异常WARN日志关掉。

**Comment by nobodyiam** [comment]
检查一下是否有配置引入了 application namespace，例如 `@EnableApolloConfig` 如果没有配置的话，默认是引入 application 的，更多可以参考 [三、客户端用法](https://www.apolloconfig.com/#/zh/usage/java-sdk-user-guide?id=%e4%b8%89%e3%80%81%e5%ae%a2%e6%88%b7%e7%ab%af%e7%94%a8%e6%b3%95)
